GETTING_STARTED.md

```diff
@@ -186,6 +186,31 @@ Another advantage of this approach is that if you need to use a `DbContext` insi
 * `WebActivatorEx`'s `PreApplicationStartMethod` (the `PreStart` method in our sample above) runs **before** OWIN's Startup method.
 	* See this StackOverflow post: https://stackoverflow.com/questions/21462777/webactivatorex-vs-owinstartup
 
+### How to deal with `ObjectDataSource` controls
+Background: `ObjectDataSource` controls invoke a `Select` method defined in the `Page` object in order to retrieve the data to bind to other controls. This select method is usually a `static` method on the containing page, which prevents using DI, but can also be an instance method, in which case it instantiates a new `Page` object and calls the method on the newly created instance. That's the scenario when DI should be possible.
+
+Unfortunately it seems that Microsoft never updated the `ObjectDataSource` control to use `HttpRuntime.WebObjectActivator`, so it's not able to instantiate DI-enabled `Page` controls. However, it is possible to override its `ObjectCreating` event to make it work the way we want:
+
+```
+using System.Web;
+using System.Web.Compilation;
+using System.Web.UI.WebControls;
+...
+public class DiObjectDataSource: ObjectDataSource 
```

Comment:
We could include this class in the project if you like (after some clean-up, of course).

Reply:
That would be great, but it has a dependency on System.Web.UI.WebControls. Do you think there should be a AspNetDependencyInjection.WebForms project like it's done for WCF ?

Reply:
`System.Web.UI.WebControls` is all in `System.Web.dll`, so it doesn't matter.

Reply:
Thanks! I force-pushed a new version of the commit that includes the control and I reworded the documentation text to reflect this. Please let me know if you have other suggestions!